### ISSUE TITLE
husky script causing errors when installing via npm

### ISSUE THREAD

**Issue by nateplusplus** [nateplusplus/pushin#93]
**Describe the bug**
When attempting to install the latest version of pushin via npm I got the following error:
```
npm ERR! command sh -c husky install
npm ERR! sh: husky: command not found
```

**To Reproduce**
1. In a new project, run `npm install pushin@latest`

**Comment by nateplusplus** [comment]
@arturovt FYI: Something seems to be up with husky.

**Comment by nateplusplus** [comment]
@arturovt I think there is still something up with this. I published version 4 to npm and I got an error in the console saying that I was not using the `publish` command correctly. It did seem to publish the package anyway though, so I tried installing the package locally from npm, however when I try running `npm install pushin@4` I see the same husky errors.

Any thoughts? Can you confirm if you get the same error?

**Comment by nateplusplus** [comment]
I have temporarily removed husky, its script and the publish scripts until we figure out the best solution.

**Comment by nateplusplus** [comment]
This is fixed in v4.0.2
